nebula-exchange/pom.xml

```diff
@@ -198,6 +202,31 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
Do we need to generate `Javadoc`?

Reply:
All Java or Scala project should provide javadoc.
Users can read the class and method through doc html.